website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -78,6 +78,8 @@ Follow the same process in [Installing dbt Cloud CLI](#manually-install-windows-
 > $ pwd
 /Users/user/dbt-projects/jaffle_shop
 
+> $ echo "project-id: '<your project id>'" > test.yml
+
 > $ cat dbt_cloud.yml
```

Comment:
what does `cat` do?

```suggestion
c. $ cat dbt_cloud.yml ## Input
```

Reply:
cat prints the contents of a file - which was not explicitly created before this PR!